react/src/components/BookImageList/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+
+import BookImage from '../BookImage';
+import {data} from '../../mock.js';
+import routes from '../../constants/routes'
+
+import './styles.css';
+
+
+class BookImageList extends Component {
+  render() {
+    let books = data.slice(0, this.props.max).map((book) => {
```

Comment:
Just a subtle comment: I'd write directly this into `{books}` curly braces inside `BookImageList`'s main `div`, so we help the garbage collector